### PR TITLE
Init homepage UI

### PR DESCRIPTION
**Pull request by Graham-Berry**
This PR Creates a homepage, an about page, and page routing logic.

### CODE REVIEW

**Review by Hodgdonc — APPROVED** [review]

**Review by sophiekummer — COMMENTED** [review]
There seem to be some issues with the functionality on MacOS that will need to be addressed before this can be merged with master.

**Review by Hodgdonc — APPROVED** [review]

**Review by Hodgdonc — CHANGES_REQUESTED** [review]
Needs linting/code formatting

**Review by Hodgdonc — APPROVED** [review]
LGTM

File: aboutPage.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        self.ldr = loader.Loader(self.logger)
+
+        self.combat_log = []
+
+        self.BuildUI()
+
+        self.UIMap = None
+
+    def BuildUI(self):
+        descText = "MAIA is a platform designed for AI competitions that provides a modular 2D\nsimulation environment for which students write AI to control competing agents.\nThe goal is to give coders all the tools necessary so that they can focus \nprimarily on analysis of information and decision-making.\n\nMAIA was developed by Dr. Zachary Hutchinson during his graduate studies at the University of Maine, Orono. Version 0.22, the most current version of MAIA, was released in October of 2020.\n\nFurther documentation, including overviews of the AI scripts, can be found in\nthe docs directory."
```
Comment (hwk01) [inline aboutPage.py]: I'm not sure how this appears on windows, but on mac some of the lines are cut out early or wraparound. 
Screen Shot 2024-02-23 at 4 44 19 PM

**Review by hwk01 — APPROVED** [review]
Everything runs perfectly well on my machine, one thing to note is when you go into the setup page if you don't put it into full screen you do not see that you need to select a team and a map.

**Pull request merged**